strawberry/unset.py

```diff
@@ -1,6 +1,22 @@
+from typing import Any
+
+
 class _Unset:
     def __str__(self):
         return ""
 
     def __bool__(self):
         return False
+
+
+UNSET: Any = _Unset()
+
+
+def is_unset(value: Any) -> bool:
+    return type(value) is _Unset
```

Comment:
do you think we can deprecate this function too and allow doing `something is UNSET`? :)

Reply:
yes this should work since all instances of `_Unset` come from this single `UNSET`.

Reply:
we should also use `Literal[UNSET]` in that case instead of `_Unset`

Reply:
where?

Reply:
https://github.com/strawberry-graphql/strawberry/blob/7d79bf9a7d7f2816bd1243f24216903c020758e3/strawberry/experimental/pydantic/utils.py#L81 and a few more times in tests

Reply:
ah yes! good idea 😊

Reply:
`Literal[UNSET]` did not work, mypy was complaining about it, I believe it has to do with the fact that `UNSET` has type `Any` (I assume so that `arg: Optional[str] = UNSET` works). So I just left it the way it was.